certified-connectors/iManage Work/apiDefinition.swagger.json

```diff
@@ -0,0 +1,6185 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "iManage Work",
+    "description": "iManage is the industry-leading provider of document and email management solutions for knowledge workers. iManage platform organizes and secures the information in documents and emails, so professionals can search for what they need, act on it, and collaborate more effectively.",
```

Comment:
@surjyams the description should contain the main value and purpose offered by the connector, please make it more precise.

Reply:
@v-EgorKozhadei Fixed